src/PAK-Samples/MapReferenceToPaper/readme.md

```diff
@@ -0,0 +1,57 @@
+# Map Academic Reference Strings to Microsoft Academic paper entities
+
+This folder contains sample code for generating academic reference strings from tabular input data and mapping the reference strings to [Microsoft Academic paper entities](https://docs.microsoft.com/en-us/academic-services/project-academic-knowledge/reference-paper-entity-attributes).
+
+## Prerequisites
+
+* Free subscription for the [Project Academic Knowledge](https://msr-apis.portal.azure-api.net/products/project-academic-knowledge) Academic Search API
+* Microsoft Visual Studio C#
+
+## Building the project
+
+1. Navigate to the MapReferenceToPaper directory
+1. Open the MapReferenceToPaper.sln file
+1. Once Visual Studio opens, build the project by going to Build->Build Solution
```

Comment:
>by going to Build->Build Solution [](start = 47, length = 33)

Suggestion:  Would it be better for them to compile the project and use the .exe that is generated as the example and document the parameters to pass?